app/Repos/UserRepo.php

```diff
@@ -145,6 +148,12 @@ public function destroy(User $user)
     {
         $user->socialAccounts()->delete();
         $user->delete();
+        
+        // Deleting User profile pics
+        $profilePic = $user->image_id ? $user->avatar->findOrFail($user->image_id) : FALSE;
```

Comment:
Here you could simply do `$profilePic = $user->avatar;` instead to achieve the same functionality and to be a bit clearer.